src/everest/util/__init__.py

```diff
@@ -76,11 +77,38 @@ def makedirs_if_needed(path, roll_if_exists=False):
     os.makedirs(path)
 
 
+def warn_user_that_runpath_is_nonempty() -> None:
+    question = (
+        "You choose to run Everest in an existing non-empty runpath.\n"
+        "Please be aware of the following:\n"
+        "- Previously generated results "
+        "might be overwritten.\n"
+        "- Previously generated files might "
+        "be used if not configured correctly.\n"
+        "Are you sure you want to continue? Y/N : "
+    )
+    if not _get_user_confirmation(question):
+        sys.exit(
+            "Everest terminated execution due to existing runpath folder.\n"
+            "Please manually delete or change simulation_folder path in the config.yml"
+        )
+
+
+def _get_user_confirmation(message: str) -> bool:
+    try:
+        result: str = input(message)
+        return bool(result.upper() in ["Y", "YES"])
+
+    except Exception as e:
+        logging.getLogger(EVEREST).error(f"Exception {e}: Unable to parse user input")
```

Comment:
Would prefix logger message with something that makes it easy for us to look it up in the AZ logs doing `startswith` search (cheaper, faster etc etc). Could be just starting the message with `Unable to parse user input`, ideally something a tiny bit more specific.

Reply:
I don't think logging this is required.